### ISSUE TITLE
1.18.2-4.0.7 Mod does not appear to function at all

### ISSUE THREAD

**Issue by Gronglegrowth** [Reforged-Mods/Terrestria-Reforged#12]
**Description of the Bug**
When loaded with Forge 40.2.4, and Terrablender 1.1.0.99 (i am on an older version because it was causing datapack errors on more recent ones, but this may have since been fixed? unsure), none of the biomes generate in the world, nor are locatable with /locatebiome, nor do any of the items and blocks appear in JEI and they cannot be given with commands. The mod does appear in the Mods menu, so it is loading, but it does not appear to be working properly?

This could be user error on my part, but I'm unsure.

**To Reproduce**
1. Install the mods with versions mentioned above.
2. Load into a world
3. Attempt, and fail, to locate one of the biomes, or obtain mod's items

**Expected behavior**
The biomes should generate in my worlds, and the items and blocks, obtainable.

**Logs & Environment**
 - Full log: [latest.log](https://github.com/Reforged-Mods/Terrestria-Reforged/files/11941912/latest.log)
 - Crash report (If available): N/A

**Comment by Trinsdar** [comment]
Do you also have traverse reforged?

**Comment by Gronglegrowth** [comment]
Yes

**Comment by Trinsdar** [comment]
ah ok, then yah, I am aware of it, haven't figured out why it happens though, it might be an issue with how forge loads jars within jars, but i'm not entirely sure about that. I have found if I drag the 3 terrestria jars within terrestria out of the jar into the mods folder, then everything works fine, for convenience I have uploaded the 3 jars as additional files on curseforge

**Comment by Lozf** [comment]
I tried using the dragging the three jars out method, and I keep running into these two crashes; forge 40.2.4, 1.18.2, terrablender 1.18.2-1.2.0.126.  I am using terrestria 4.0.7.1, although it also occurs with terrestria 4.0.7It happens a majority of the time, although sometimes it does load.  I tried to see if it was a conflict in my mod list but wasn't able to narrow anything down.  The crashes do not seem to occur when the mods are on their own.

Crash for the boat: https://gist.github.com/Lozf/e4b9f6e013ac8076ca96c84308fda61d
latest log for the boat: https://gist.github.com/Lozf/bb5c6814131e2e5cb64368eaaa8fb370

Crash for terraform wood API: https://gist.github.com/Lozf/0c390b54a4dcc0f9ba8e1a06d0d8ec04
latest log for terraform wood API: https://gist.github.com/Lozf/e41e5d872875a8b6323ac289196cf79d

The oddest thing is it used to work with traverse and terrestria loading just fine without needing to drag out the three jars, although now it acts like above, with terrestria not functioning.
